app/bundles/AssetBundle/Model/AssetModel.php

```diff
@@ -159,6 +159,11 @@ public function trackDownload($asset, $request = null, $code = '200', $systemEnt
 
         $download = new Download();
         $download->setDateDownload(new \Datetime());
+        $download->setUtmCampaign($request->get('utm_campaign') ?? null);
+        $download->setUtmContent($request->get('utm_content') ?? null);
+        $download->setUtmMedium($request->get('utm_medium') ?? null);
+        $download->setUtmSource($request->get('utm_source') ?? null);
+        $download->setUtmTerm($request->get('utm_term') ?? null);
```

Comment:
Setting default value of `null` like this is not necessary as the get method returns `null` as the default value automatically.
```suggestion
        $download->setUtmCampaign($request->get('utm_campaign'));
        $download->setUtmContent($request->get('utm_content'));
        $download->setUtmMedium($request->get('utm_medium'));
        $download->setUtmSource($request->get('utm_source'));
        $download->setUtmTerm($request->get('utm_term'));
```